localenv/mock-account-servicing-entity/app/lib/webhooks.server.ts

```diff
@@ -157,6 +160,70 @@ export async function handleIncomingPaymentCompletedExpired(wh: Webhook) {
   return
 }
 
+export async function handleWalletAddressWebMonetization(wh: Webhook) {
+  const walletAddressUrl = wh.data['walletAddressUrl'] as string | undefined
+
+  if (!walletAddressUrl) {
+    throw new Error('No walletAddressUrl found')
+  }
+
+  const accountPath = new URL(walletAddressUrl).pathname.substring(1)
+  const account = await mockAccounts.getByPath(accountPath)
+  const withdrawal = wh.data
+  const wa = withdrawal['walletAddressId'] as string
+  const withdrawalId = withdrawal['id'] as string
+
+  if (!account) {
+    throw new Error('No account found for wallet address')
+  }
+
```

Comment:
The call to actually credit the amount is missing (`mockAccounts.credit`). See `handleIncomingPaymentCompletedExpired` for inspiration ;-)

Reply:
In addition to @sabineschaller 's comment, we should

1. call the `CreateWalletAddressWithdrawal` mutation (this will create a pending withdrawal)
2. Call `mockAccounts.credit` to actually credit the account the incoming value
3. Call `postLiquidityWithdrawal` mutation (to confirm the withdrawal)

(Right @sabineschaller? since we saw that the `CreateWalletAddressWithdrawal` passes in a `timeout` to `accountingService.createWithdrawal` we should already make this one a proper two-phase transfer, like all of the other liquidity mutations in the future)

Reply:
Correct. I added my comment prematurely.